docker-compose.yml

```
@@ -1,40 +1,19 @@
-version: '2'
+version: '3.8'
 
 services:
   ws:
-    image: navitia/chaos-ws:dev
+    image: chaos-prod:1.0.0
```

Why `chaos-prod` ?
Image are already on the official docker hub. https://hub.docker.com/r/navitia/chaos-ws 

```suggestion
    image: navitia/chaos-ws:${VERSION}
```

I think version must be a variable for devops and deploy.